generators/app/templates/src/index.js

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
 
-import App from './App';
+import App from './app';
+import registerServiceWorker from './registerServiceWorker';
 import './config/i18n';
+import './scss/application.scss';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const render = () => {
+  ReactDOM.render(
+    <AppContainer>
+      <App />
+    </AppContainer>,
+    document.getElementById('root')
+  );
+};
+
+// Render once
+render(App);
```

Comment:
We aren't using this param

Reply:
Why we need to render this once ? Does this relate with WHMR API ?

Reply:
The documentation indicates that it should be implemented in this way
https://github.com/gaearon/react-hot-loader#api

Reply:
Then I'd generalize `render` function to receive a `Component`. By now, it doesn't receive any param 😊